test/pypi-integration.py

```diff
@@ -34,6 +34,7 @@ def test_get_pypi(self):
                 'source': {
                     'name': 'numpy',
                     'test': False,
+                    'python_version': '3.7',
```

Comment:
This `numpy` is not available for Python >=`3.9` so it was either this or bumping the `numpy` version. Since this solution will always work, I decided for this option. Bumping the version can potentially break in future in the same way.